int8 getQ
---
float64[6] Q
